rules/linux/command_and_control_connection_accept_by_non_ssh_root_sesssion.toml

```
@@ -0,0 +1,80 @@
+[metadata]
+creation_date = "2022/05/17"
+maturity = "production"
+updated_date = "2022/05/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a session id change followed by an accepted network connection in correlatioin with a ptmx file creation and the process that accepted the connection ending all as user root by the same process
```

`python -m detection_rules toml-lint -f` or line break at 120

---

Can you rework this a bit. It explains the query well, but not so much the actual activity

```suggestion
Identifies a session id change followed by an incoming network connection in correlation with a ptmx file creation and the process that accepted the connection ending all as user root by the same process
```